src/Core/Repositories/EntityFramework/DatabaseContext.cs

```diff
@@ -10,10 +10,29 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options)
             : base(options)
         { }
 
-        public DbSet<User> Users { get; set; }
         public DbSet<Cipher> Ciphers { get; set; }
+        public DbSet<Collection> Collections { get; set; }
+        public DbSet<CollectionCipher> CollectionCiphers { get; set; }
+        public DbSet<Device> Devices { get; set; }
+        public DbSet<EmergencyAccess> EmergencyAccesses { get; set; }
+        public DbSet<Event> Events { get; set; }
+        public DbSet<Folder> Folders { get; set; }
+        public DbSet<Grant> Grants { get; set; }
+        public DbSet<Group> Groups { get; set; }
+        public DbSet<GroupUser> GroupUsers { get; set; }
+        public DbSet<Installation> Installations { get; set; }
         public DbSet<Organization> Organizations { get; set; }
-
+        public DbSet<OrganizationUser> OrganizationUsers { get; set; }
+        public DbSet<Policy> Policies { get; set; }
+        public DbSet<Role> Roles { get; set; }
+        public DbSet<Send> Sends { get; set; }
+        public DbSet<SsoConfig> SsoConfigs { get; set; }
+        public DbSet<SsoUser> SsoUsers { get; set; }
+        public DbSet<TaxRate> TaxRates { get; set; }
+        public DbSet<Transaction> Transactions { get; set; }
+        public DbSet<U2f> U2fs { get; set; }
+        public DbSet<User> Users { get; set; }
+        
```

Comment:
Do we need to add some stuff to `OnModelCreating` below?

Reply:
I don't have any provider-specific changes set up yet in the models, but I went ahead and added the table mappings for relational dbs.